hereditary/ontology/genomics/.htaccess

```
@@ -33,6 +27,6 @@ RewriteRule ^$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_geno
 RewriteRule ^$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_genomics.ttl [R=303]
 
 # Rewrite rule to deal with two namespaces: schema and resource
-RewriteRule ^schema/(.*)$ http://hereditary.dei.unipd.it/ontology/genomics/schema/#$1 [R=302,L,NE]
-RewriteRule ^resource/(.*)$ http://hereditary.dei.unipd.it/ontology/#https://w3id.org/hereditary/ontology/genomics/resource/$1 [R=302,L,NE]
+RewriteRule ^/schema/(.*)$ http://hereditary.dei.unipd.it/ontology/genomics/schema/#$1 [R=302,L,NE]
```

```suggestion
RewriteRule ^schema/(.*)$ http://hereditary.dei.unipd.it/ontology/genomics/schema/#$1 [R=302,L,NE]
```
Leave it as it was, it was correct